pwa-devdocs/src/tutorials/cloud-deploy/index.md

```diff
@@ -175,44 +138,26 @@ These commands modify the `composer.json` file and adds the sample data modules
 "magento/sample-data-media-venia": "dev-master",
 ```
 
-## Modify `.gitignore` file
-
-Add the following frontend dependency entries to the `.gitignore` file to track them in your repository.
-
-```text
-!package.json
-!yarn.lock
-```
-
 ## Install dependencies
 
-### Frontend dependencies
-
-Install the frontend dependencies specified in the `package.json` file.
-
-```sh
-yarn install
-```
-
-A successful installation creates a `yarn.lock` file in your project directory.
-
 ### Composer dependencies
 
-Install the new Composer dependencies added to the `composer.json` file.
+_Optional_: If you manually modified the `composer.json` file to make the changes described in the previous steps, update your `composer.lock` with these new dependencies:
 
 ```sh
 composer update
 ```
 
 ## Specify Cloud server environment
 
-### Install Yarn
+### Install NPX and Yarn
 
-If your project uses Yarn, which is the case for `venia-concept`, add the following entry to the `.magento.app.yaml` file.
+If your project supports Yarn, which is the case for `venia-concept`, add the following entry to the `.magento.app.yaml` file.
 
 ```yaml
 dependencies:
     nodejs:
+      npx: "*"
```

Comment:
Did you find that `npx` actually needs to be installed? On recent versions of Node it comes built-in.

Reply:
Doesn't seem so, must have got left behind when experimenting with removal of NVM steps. Removed explicit dependency in f934608.